Rubberduck.Refactorings/MoveCloserToUsage/MoveCloserToUsageRefactoringAction.cs

```diff
@@ -16,12 +17,17 @@ public MoveCloserToUsageRefactoringAction(IRewritingManager rewritingManager)
         protected override void Refactor(MoveCloserToUsageModel model, IRewriteSession rewriteSession)
         {
             var target = model.Target;
-            InsertNewDeclaration(target, rewriteSession);
-            RemoveOldDeclaration(target, rewriteSession);
-            UpdateQualifiedCalls(target, rewriteSession);
+            if (target is VariableDeclaration variable)
```

Comment:
That should probably be inverted. The result is a guard clause and no need to use `return`.